backend/config.js

```diff
@@ -0,0 +1,135 @@
+module.exports = {
```

Comment:
Please rename this to `config-dev.js` or something and add a comment in `README.txt` to `cp config-dev.js config.js`